Misc/NEWS.d/next/Library/2025-06-25-14-13-39.gh-issue-135661.idjQ0B.rst

```diff
@@ -0,0 +1,23 @@
+Fix parsing start and end tags in :class:`html.parser.HTMLParser`.
+
+* Whitespaces no longer accepted between ``</`` and the tag name.
+  E.g. ``</ script>`` does not end the script section.
+
+* Vertical tabulation (``\v``) and non-ASCII whitespaces no longer recognized
+  as whitespaces. The only whitespaces are ``\t\n\r\f`` and space.
+
+* Null character (U+0000) no longer ends the tag name.
+
+* End tag can have attributes and slashes after tag name. It no longer ends
+  after the first ``>`` in quoted attribute value. E.g. ``</script/foo=">"/>``.
```

Comment:
This seems to suggest that they are valid -- I would rephrase it to something like:
```suggestion
* Attributes and slashes after the tag name in end tags are now correctly
  parsed as comments, instead of terminating after the first ``>``
  in quoted attribute value. E.g. ``</script/foo=">"/>``.
```

Reply:
Actually, they are ignored.